db/migrate/20221112175850_create_fruit_ticket_transactions.rb

```diff
@@ -0,0 +1,15 @@
+class CreateFruitTicketTransactions < ActiveRecord::Migration[5.2]
+  def change
+    create_table :fruit_ticket_transactions do |t|
+      # t.references :user, null: false, index: true
+      t.integer :transaction_type, null: false
+      t.integer :source_id
+
+      t.integer :amount, null: false, default: 0
+      t.integer :from_user_id
+      t.integer :to_user_id
```

Comment:
Doesn't seem good that `to_user_id` and `from_user_id` are both nullable, seems at least one of them should be `null: false`.

Reply:
hmmm yeah, maybe `to_user_id` shouldn't be nullable. from my understanding, the transaction should always have a destination.

Reply:
Well for example when you do a fruit summon the from_user is who did the fruit summon and there is no to_user. 
When supporters are given the monthly fruit ticket stipend, the user who receives the tickets is the to_user and there is no from_user.

Seems hacky but I was thinking in the case of no to/from user, the user could just be designated as 0, meaning 'the website' was the from/to user.

Reply:
I added a validation but would be nice to enforce it at the db level somehow. https://github.com/datafruits/streampusher-api/pull/191/files#diff-54996b8ab9034d59761bf5d32a9dcc547c75ce4a5a0a8c6695ffb4fc5d9d6131R52

Reply:
thanks for explaining this to me :pray: The validation looks right to me